Source/Scene/ModelExperimental/Model.js

```diff
@@ -1156,7 +1191,6 @@ Object.defineProperties(Model.prototype, {
    * @memberof Model.prototype
    *
    * @type {String}
-   * @experimental This feature is using part of the 3D Tiles spec that is not final and is subject to change without Cesium's standard deprecation policy.
```

Comment:
I would keep this `@experimental` tag for this and `instanceFeatureIdLabel` below. someday when #10025 is addressed, this detail might change.

Reply:
I also swept through the other properties, I'd also add the experimental tag for `customShader`